src/sentry/templates/sentry/debug/mail/preview.html

```diff
@@ -53,6 +53,9 @@
         <option value="mail/sentry-app-notify-disable/">Sentry App Notify Disable</option>
         <option value="mail/join-request/">Join Request</option>
       </optgroup>
+      <optgroup label="Relocation">
+        <option value="mail/relocation-started/">Relocation Started</option>
```

Comment:
should we move `Relocate Account` here if this is now a new group?

Reply:
Done.